content/asp.net/getting-started/files.md

```diff
@@ -3,13 +3,76 @@ by [Steve Smith](http://deviq.com/me/steve-smith)
 
 See the [Issue](https://github.com/dotnet/training-tutorials/issues/48) to claim this lesson and/or view what should be included in it.
 
-## First Header
+## Program.cs
 
-Start the lesson here.
+In the [first lesson](your-first.md), you saw how to take a console application and convert it into an ASP.NET Core web app. Part of that process involved updating the ``Main`` method in *Program.cs* to configure and run a *host*. ASP.NET apps must run within a host, and this host is typically configured in a ``Main`` method. Once configured, you won't typically need to update this file very often going forward. A very simple, minimal example would be:
 
-...
+```c#
+using System;
+using Microsoft.AspNetCore.Hosting;
 
+namespace ConsoleApplication
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var host = new WebHostBuilder()
+                .UseKestrel()
+                .UseStartup<Startup>()
+                .Build();
+
+            host.Run();
+        }
+    }
+}
+```
+
+Note that the above class references the ``Startup`` type, which is where you configure your individual app.
```

Comment:
If this is an intro *intro* **intro** level thing, should we link out to generics, or mention that the call is generic and accepts a type, but that we'll look more into that later?